packages/es-components/src/components/patterns/datepicker/DatePicker.js

```diff
@@ -56,7 +56,11 @@ const DateTextbox = React.forwardRef(function DateTextbox(props, ref) {
 
   const textbox = (
     <MaskedTextbox
+      css="
+        width: 265px;
```

Comment:
Should we use max-width here in case the picker is in a smaller parent element?

Reply:
I think that makes sense, I'll make that change.